### DISCUSSION TITLE
Display file size in Gallery file listing

### DISCUSSION THREAD

**Discussion by mikegilchrist** [FossifyOrg/General-Discussion#258] [Ideas]
Perhaps I'm missing a setting somewhere, but it would be nice to be able to see the size of the file in the list and grid view.  I'd even be okay with it if it required shortening the displayed name since most files include very fine scale timestamps in their names.

**Comment by Aga-C** [comment]
Please create a feature request in [Gallery](https://github.com/FossifyOrg/Gallery/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml) for it.
